client/my-sites/plugins/plugins-browser/style.scss

```diff
@@ -80,6 +80,8 @@
 }
 
 body.is-section-plugins #primary .main.is-logged-out {
+	// Need to specifically set this, to override the classic blue.
+	--wp-admin-theme-color: var(--studio-blue-50);
```

Comment:
Just curious, where is the classic blue coming from, and can we eliminate that at the souce? I'm guessing from WordPress itself so _no_, just making sure.

Reply:
I tried so hard to do that 😅  It seems to be a CSS loading order issue. It does come from the core component base styles, but usually it gets overridden by the color-studio definition. That works on the logged out theme showcase, but for some reason on logged out plugins, the core definition "wins".

Reply:
CC: @ciampo @mirka in case you have insights.

Reply:
For the sake of this PR, is the override okay to achieve the purpose until the package versions can be looked at?

Reply:
Yes.